packages/fxa-settings/src/components/BrandMessaging/index.tsx

```diff
@@ -68,7 +68,10 @@ const BrandMessaging = ({
   }
 
   return (
-    <div id="banner-brand-message" className="fixed w-full top-0 left-0">
+    <div
+      id="banner-brand-message"
+      className="w-full relative mobileLandscape:absolute mobileLandscape:top-0 mobileLandscape:left-0"
```

Comment:
`left-0` should perhaps be `start-0` to account for RTL

Reply:
This is just a box. I think in this case putting the box full width at 0,0, is okay.